AISKU/rollup.config.js

```diff
@@ -31,7 +32,9 @@ const browserRollupConfigFactory = (isProduction, libVersion = '2') => {
       nodeResolve({
         browser: false,
         preferBuiltins: false
-      })
+      }),
+      es3Poly(),
```

Comment:
New rollup plugins that replace some of the incompatible ES3 code and check that we are not including some of the easily detectable known methods.